tst/elements.tst

```diff
@@ -14,7 +14,7 @@ gap> a :=l[1]; b:=l[2]; # a and b are images of group generators in KG
 (Z(2)^0)*f2
 gap> e := One( KG );    # for convenience, we denote the identity by e
 (Z(2)^0)*<identity> of ...
-gap> x := ( e + a ) * ( e + b ); 
+gap> x := ( e + a ) * ( e + b );
```

Comment:
This is apparently automatically changed by Atom editor. We need to check its settings, as this may cause a trouble later.

Reply:
https://flight-manual.atom.io/faq/sections/why-is-atom-deleting-trailing-whitespace-why-is-there-a-newline-at-the-end-of-the-file/ explains that "Atom ships with the whitespace package, which by default strips trailing whitespace from lines in your file, and inserts a final trailing newline to indicate end-of-file as per the POSIX standard". See the link for the instructions how to disable this behaviour.